circuits/demo-circuit.circom

```diff
@@ -1,11 +1,11 @@
 pragma circom 2.0.0;
```

Comment:
Output of this auto-generated circuit would be a prediction by the model for some testing data (i.e. testing image of size of 28 * 28 * 1, in this case). Should we also prove that a prediction is correct using another custom circuit, which takes testing label and output of this circuit (model prediction) as inputs, and outputs assertion of the prediction and testing label?

Reply:
This is also out of scope rn.